packages/@ember/-internals/glimmer/lib/components/link-to.ts

```diff
@@ -406,6 +406,18 @@ const LinkComponent = EmberComponent.extend({
   **/
   replace: false,
 
+  /**
+    Determines whether the `LinkComponent` will prevent the default
```

Comment:
Based on the description of the `Allowing Default Action` section in [the LinkTo API docs](https://api.emberjs.com/ember/release/classes/Ember.Templates.components/methods/LinkTo?anchor=LinkTo).

Reply:
Now that I think about it, I wonder if this should just be deprecated and removed as part of RFC 707.

This seems only useful in the context of changing the tagName to something other than `<a>` (which is deprecated in RFC 707).

When the tagName is `<a>`, the default action of the browser is to navigate away, so there is no sense in not preventing it.

Thoughts?

Looping in @rwjblue as well.

Reply:
I'm not sure I completely understand what you mean. I've never used `@preventDefault` myself, but I can imagine it being useful for some use cases to open a route in a new tab.

Reply:
Wow sorry, scratch that. I was confused. Yeah I can't immediately imagine a scenario where a refresh is desired.

Reply:
@chancancode - ya seems correct to me